pyvisa/util.py

```diff
@@ -1057,3 +1057,32 @@ def get_arch(filename: Union[str, Path]) -> List[ArchitectureType]:
             archs.append(ArchitectureType.AARCH64)
 
         return archs
+
+
+def message_size(
+    num_points: int,
+    point_size: Literal[1, 2] = 2,
+    header_format: Literal["ieee", "hp", "empty"] = "ieee",
+) -> int:
+    """Helper function to calculate a message size, including the header, in bytes.
+
+    Inputs:
+        num_points: Number of data points to transfer
+        point_size: Size of each data point in bytes
+        header_format: Specify "ieee" or "hp" or "empty" header format
+
+    Returns:
+        The total message size in bytes
```

Comment:
The library use the Numpy-Style for docstrings for I would prefer to remain consistent
```suggestion
    Parameters
    -----------
    num_points : int
        Number of data points to transfer
    point_size : int
        Size of each data point in bytes
    header_format : str
        Specify "ieee" or "hp" or "empty" header format

    Returns
    --------
    int
        The total message size in bytes
```

Reply:
Done/